src/ort_genai_c.h

```diff
@@ -151,15 +163,47 @@ OGA_EXPORT OgaResult* OGA_API_CALL OgaGenerate(const OgaModel* model, const OgaG
  */
 OGA_EXPORT OgaResult* OGA_API_CALL OgaCreateGeneratorParams(const OgaModel* model, OgaGeneratorParams** out);
 
+/*
+ * \brief Creates Lora Adapter within a given model that one can add
+ *        parameters to. The adapter_name must be unique within the model.
+ * \param[in] lora_manager where to create the adapter
+ * \param[in] adapter_name to be created
+ * \return OgaResult containing the error message if the adapter creation failed.
+ */
+OGA_EXPORT OgaResult* OGA_API_CALL OgaCreateLoraAdapter(OgaLoraManagerInternal* lora_manager, const char* adapter_name);
```

Comment:
Do we need to provide an API to list all the available adapters?

Reply:
The process is config driven. And, therefore, the adapters created and used will be known beforehand. I feel the API would be useless, and we could add it as needed.

Reply:
I propose to put lora things into the config and get rid of LoraManager API